external/README.md

```diff
@@ -11,7 +11,7 @@ The current published artifacts are:
 * CLI Download Links:
   * [MacOS](https://bintray.com/lightbend/cloudflow-cli/download_file?file_path=kubectl-cloudflow-1.3.0-M1.37-1e4b7f3-darwin-amd64.tar.gz)
   * [Linux (amd64 arch)](https://bintray.com/lightbend/cloudflow-cli/download_file?file_path=kubectl-cloudflow-1.3.0-M1.37-1e4b7f3-linux-amd64.tar.gz)
-  * [Win (64 bits)] (https://bintray.com/lightbend/cloudflow-cli/download_file?file_path=kubectl-cloudflow-1.3.0-M1.37-1e4b7f3-windows-amd64.tar.gz)
+  * [Win (64 bits)](https://bintray.com/lightbend/cloudflow-cli/download_file?file_path=kubectl-cloudflow-1.3.0-M1.37-1e4b7f3-windows-amd64.tar.gz)
```

Comment:
Why do we have the CLI download links in the "external" folder READMe anyway?We're for sure going to forget to update those download links after a release...

Reply:
@agemooij this page historically summarized our artifacts. Do you want me to remove them?

Reply:
@skonto I think it is not sustainable to manually maintain this page and I'm also not sure who the audience for this information is. Let's discuss whether we need this page and, if we do, how we can generate it as part of the release.

Reply:
@agemooij the audience for this is for who ever want to modify our images with custom stuff or any other contributor.

Reply:
Then the CLI download links should definitely not be in here since that is unrelated and not meant for customization

Reply:
@agemooij ok I will remove the cli links.